bugsnag-android-core/src/main/java/com/bugsnag/android/Stackframe.kt

```diff
@@ -40,8 +40,20 @@ class Stackframe @JvmOverloads internal constructor(
     var columnNumber: Number? = null
 ) : JsonStream.Streamable {
 
+    private var nativeFrame: NativeStackframe? = null
+
+    constructor(nativeFrame: NativeStackframe): this(nativeFrame.method, nativeFrame.file, nativeFrame.lineNumber, false, null) {
```

Comment:
I think the arguments should probably be null for this, as the design's requirement is that the properties would not be publicly accessible. Furthermore if a user edited the `method` with this approach, the change would not be reflected in the payload sent to bugsnag - which would be confusing behaviour

Reply:
This is where the model starts to break down... I'm not so sure that Stackframe is the right holder for the native frame, since it's essentially lying to the outside world via its public interface.

Reply:
Updated to propagate the setters to the wrapped native trace if it's available.